Sources/SwiftDocC/Model/Rendering/Symbol/DeclarationsRenderSection.swift

```diff
@@ -114,24 +114,40 @@ public struct DeclarationRenderSection: Codable, Equatable {
         
         /// If the token is a known symbol, its precise identifier as vended in the symbol graph.
         public let preciseIdentifier: String?
-        
+
+        /// The kind of highlight the token should be rendered with.
+        public let highlight: Highlight?
+
+        /// The kinds of highlights that can be applied to a token.
+        public enum Highlight: String, Codable, RawRepresentable {
+            /// A highlight representing generalized change, not specifically added or removed.
+            case changed
```

Comment:
Do we ever imagine other types of highlights? Would this be simpler as a boolean? You mention "added" or "removed" which seem like good cases to add here, but the code doesn't use those concepts yet? Are you planning to add them later?

Reply:
Yes; Swift-DocC-Render currently has `added` and `removed` token kinds that use the recursive structure we talked about earlier. The idea is that we want to migrate that to this pattern eventually, but only once Swift-DocC is able to properly emit them.